R/git.R

```diff
@@ -449,7 +449,7 @@ use_git_credentials <- function(credentials) {
 #'
 #' Get a situation report on your current git/GitHub status. Useful for
 #' diagnosing problems.
-#'
+#' To vaccinate (have a general git ignore file) use [edit_git_ignore()]
```

Comment:
This should refer people to the `git_vaccinate()` function.

Reply:
I didn't even knew that this function existed.